Hm, so 7.0.0 fails to compile:

<details>
```
cc1: some warnings being treated as errors
make[2]: *** [CMakeFiles/multitail.dir/build.make:314: CMakeFiles/multitail.dir/stripstring.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/build/source/utils.c: In function 'amount_to_str':
/build/source/utils.c:373:52: warning: '__builtin___snprintf_chk' output may be truncated before the last format character [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-truncation=-Wformat-truncation=8;;]
  373 |                 snprintf(out, AMOUNT_STR_LEN, "%dKB", (int)((amount + M_KB - 1) / M_KB));
      |                                                    ^
In file included from /nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/stdio.h:906,
                 from /build/source/utils.c:7:
/nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/bits/stdio2.h:54:10: note: '__builtin___snprintf_chk' output between 4 and 7 bytes into a destination of size 6
   54 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   55 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   56 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
/build/source/utils.c:371:52: warning: '__builtin___snprintf_chk' output may be truncated before the last format character [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-truncation=-Wformat-truncation=8;;]
  371 |                 snprintf(out, AMOUNT_STR_LEN, "%dMB", (int)((amount + M_MB - 1) / M_MB));
      |                                                    ^
In file included from /nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/stdio.h:906,
                 from /build/source/utils.c:7:
/nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/bits/stdio2.h:54:10: note: '__builtin___snprintf_chk' output between 4 and 7 bytes into a destination of size 6
   54 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   55 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   56 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
cc1: some warnings being treated as errors
make[2]: *** [CMakeFiles/multitail.dir/build.make:328: CMakeFiles/multitail.dir/term.c.o] Error 1
/build/source/ui.c: In function 'set_window_sizes':
/build/source/ui.c:2168:83: warning: '%d' directive output may be truncated writing between 1 and 10 bytes into a region of size 5 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-truncation=-Wformat-truncation=8;;]
 2168 |                                                 snprintf(oldval, sizeof(oldval), "%d", pi[window].win_height);
      |                                                                                   ^~
/build/source/ui.c:2168:82: note: directive argument in the range [0, 2147483647]
 2168 |                                                 snprintf(oldval, sizeof(oldval), "%d", pi[window].win_height);
      |                                                                                  ^~~~
In file included from /nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/stdio.h:906,
                 from /build/source/ui.c:5:
/nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/bits/stdio2.h:54:10: note: '__builtin___snprintf_chk' output between 2 and 11 bytes into a destination of size 5
   54 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  cc1: some warnings being treated as errors
make[2]: *** [CMakeFiles/multitail.dir/build.make:314: CMakeFiles/multitail.dir/stripstring.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/build/source/utils.c: In function 'amount_to_str':
/build/source/utils.c:373:52: warning: '__builtin___snprintf_chk' output may be truncated before the last format character [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-truncation=-Wformat-truncation=8;;]
  373 |                 snprintf(out, AMOUNT_STR_LEN, "%dKB", (int)((amount + M_KB - 1) / M_KB));
      |                                                    ^
In file included from /nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/stdio.h:906,
                 from /build/source/utils.c:7:
/nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/bits/stdio2.h:54:10: note: '__builtin___snprintf_chk' output between 4 and 7 bytes into a destination of size 6
   54 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   55 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   56 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
/build/source/utils.c:371:52: warning: '__builtin___snprintf_chk' output may be truncated before the last format character [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-truncation=-Wformat-truncation=8;;]
  371 |                 snprintf(out, AMOUNT_STR_LEN, "%dMB", (int)((amount + M_MB - 1) / M_MB));
      |                                                    ^
In file included from /nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/stdio.h:906,
                 from /build/source/utils.c:7:
/nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/bits/stdio2.h:54:10: note: '__builtin___snprintf_chk' output between 4 and 7 bytes into a destination of size 6
   54 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   55 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   56 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
cc1: some warnings being treated as errors
make[2]: *** [CMakeFiles/multitail.dir/build.make:328: CMakeFiles/multitail.dir/term.c.o] Error 1
/build/source/ui.c: In function 'set_window_sizes':
/build/source/ui.c:2168:83: warning: '%d' directive output may be truncated writing between 1 and 10 bytes into a region of size 5 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-truncation=-Wformat-truncation=8;;]
 2168 |                                                 snprintf(oldval, sizeof(oldval), "%d", pi[window].win_height);
      |                                                                                   ^~
/build/source/ui.c:2168:82: note: directive argument in the range [0, 2147483647]
 2168 |                                                 snprintf(oldval, sizeof(oldval), "%d", pi[window].win_height);
      |                                                                                  ^~~~
In file included from /nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/stdio.h:906,
                 from /build/source/ui.c:5:
/nix/store/p5vg4rdba95snvxd7p0km695jnm8vvwh-glibc-2.35-163-dev/include/bits/stdio2.h:54:10: note: '__builtin___snprintf_chk' output between 2 and 11 bytes into a destination of size 5
   54 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   55 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   56 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
cc1: some warnings being treated as errors
make[2]: *** [CMakeFiles/multitail.dir/build.make:342: CMakeFiles/multitail.dir/ui.c.o] Error 1
cc1: some warnings being treated as errors
make[2]: *** [CMakeFiles/multitail.dir/build.make:258: CMakeFiles/multitail.dir/mt.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/multitail.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
error: builder for '/nix/store/vrqib031xh33mcfjqjdzca1d4hmmcm9l-multitail-7.0.0.drv' failed with exit code 2;
       last 10 log lines:
       >    55 |                                    __glibc_objsize (__s), __fmt,
       >       |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       >    56 |                                    __va_arg_pack ());
       >       |                                    ~~~~~~~~~~~~~~~~~
       > cc1: some warnings being treated as errors
       > make[2]: *** [CMakeFiles/multitail.dir/build.make:342: CMakeFiles/multitail.dir/ui.c.o] Error 1
       > cc1: some warnings being treated as errors
       > make[2]: *** [CMakeFiles/multitail.dir/build.make:258: CMakeFiles/multitail.dir/mt.c.o] Error 1
       > make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/multitail.dir/all] Error 2
       > make: *** [Makefile:136: all] Error 2
       For full logs, run 'nix log /nix/store/vrqib031xh33mcfjqjdzca1d4hmmcm9l-multitail-7.0.0.drv'.
 1 update-multitail | 
 55 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   56 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
cc1: some warnings being treated as errors
make[2]: *** [CMakeFiles/multitail.dir/build.make:342: CMakeFiles/multitail.dir/ui.c.o] Error 1
cc1: some warnings being treated as errors
make[2]: *** [CMakeFiles/multitail.dir/build.make:258: CMakeFiles/multitail.dir/mt.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/multitail.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
error: builder for '/nix/store/vrqib031xh33mcfjqjdzca1d4hmmcm9l-multitail-7.0.0.drv' failed with exit code 2;
       last 10 log lines:
       >    55 |                                    __glibc_objsize (__s), __fmt,
       >       |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       >    56 |                                    __va_arg_pack ());
       >       |                                    ~~~~~~~~~~~~~~~~~
       > cc1: some warnings being treated as errors
       > make[2]: *** [CMakeFiles/multitail.dir/build.make:342: CMakeFiles/multitail.dir/ui.c.o] Error 1
       > cc1: some warnings being treated as errors
       > make[2]: *** [CMakeFiles/multitail.dir/build.make:258: CMakeFiles/multitail.dir/mt.c.o] Error 1
       > make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/multitail.dir/all] Error 2
       > make: *** [Makefile:136: all] Error 2
       For full logs, run 'nix log /nix/store/vrqib031xh33mcfjqjdzca1d4hmmcm9l-multitail-7.0.0.drv'.
 1 update-multitail | 
```
</details>

I am pretty sure that this is not our issue, is it?

--- a/pkgs/tools/misc/multitail/default.nix
+++ b/pkgs/tools/misc/multitail/default.nix
@@ -1,17 +1,17 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, ncurses, pkg-config }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, ncurses, pkg-config, cmake }:
 
 stdenv.mkDerivation rec {
-  version = "6.5.2";
+  version = "7.0.0";
   pname = "multitail";
 
   src = fetchFromGitHub {
-    owner = "halturin";
+    owner = "folkertvanheusden";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-DMYcoHOaX4bToDE8qpXmg5tbTYGkiwtPJd0/qy4uD54=";
+    rev = version;
+    sha256 = "sha256-AMW55Bmwn0BsD36qGXI5WmEfydrMBob8NRY3Tyq92vA=";
   };
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config cmake ];
 
   buildInputs = [ ncurses ];
 
@@ -19,8 +19,10 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
-    cp multitail $out/bin
+    cp bin/multitail $out/bin
   '';
+
+  hardeningDisable = [ "format" ];
 
   meta = {
     homepage = "https://github.com/halturin/multitail";

